install/aws_policy.json

```diff
@@ -1,174 +1,158 @@
-                "iam:DetachRolePolicy",
-                "iam:GetInstanceProfile",
-                "iam:GetPolicy",
-                "iam:GetPolicyVersion",
-                "iam:GetRole",
-                "iam:ListEntitiesForPolicy",
-                "iam:ListInstanceProfilesForRole",
-                "iam:ListPolicyVersions",
-                "iam:PassRole",
-                "iam:RemoveRoleFromInstanceProfile",
-                "route53:ChangeResourceRecordSets",
-                "route53:CreateHostedZone",
-                "route53:DeleteHostedZone",
-                "route53:GetChange",
-                "route53:GetHostedZone",
-                "route53:ListHostedZones",
-                "route53:ListResourceRecordSets",
-                "route53:ListTagsForResource",
-                "s3:CreateBucket",
-                "s3:DeleteBucket",
-                "s3:DeleteObject",
-                "s3:DeleteObjectTagging",
-                "s3:DeleteObjectVersion",
-                "s3:DeleteObjectVersionTagging",
-                "s3:GetAccelerateConfiguration",
-                "s3:GetAnalyticsConfiguration",
-                "s3:GetBucketAcl",
-                "s3:GetBucketCORS",
-                "s3:GetBucketLocation",
-                "s3:GetBucketLogging",
-                "s3:GetBucketNotification",
-                "s3:GetBucketObjectLockConfiguration",
-                "s3:GetBucketPolicy",
-                "s3:GetBucketPolicyStatus",
-                "s3:GetBucketPublicAccessBlock",
-                "s3:GetBucketRequestPayment",
-                "s3:GetBucketTagging",
-                "s3:GetBucketVersioning",
-                "s3:GetBucketWebsite",
-                "s3:GetEncryptionConfiguration",
-                "s3:GetInventoryConfiguration",
-                "s3:GetLifecycleConfiguration",
-                "s3:GetMetricsConfiguration",
-                "s3:GetObject",
-                "s3:GetObjectAcl",
-                "s3:GetObjectLegalHold",
-                "s3:GetObjectRetention",
-                "s3:GetObjectTagging",
-                "s3:GetObjectTorrent",
-                "s3:GetObjectVersion",
-                "s3:GetObjectVersionAcl",
-                "s3:GetObjectVersionForReplication",
-                "s3:GetObjectVersionTagging",
-                "s3:GetObjectVersionTorrent",
-                "s3:GetReplicationConfiguration",
-                "s3:ListBucket",
-                "s3:ListBucketMultipartUploads",
-                "s3:ListBucketVersions",
-                "s3:ListMultipartUploadParts",
-                "s3:PutBucketAcl",
-                "s3:PutBucketVersioning",
-                "s3:PutObject",
-                "s3:PutObjectAcl",
-                "s3:PutObjectTagging",
-                "s3:PutObjectVersionAcl",
-                "s3:PutObjectVersionTagging",
-                "sts:AssumeRole",
-                "sts:GetCallerIdentity"
-            ],
-            "Resource": "*"
-        }
-    ]
```

Comment:
This is policy optimization? 
Are you test it with Minikube?

Reply:
Yes it was tested with minikube, It also in active testing now. Found some extra issues